src/main/kotlin/com/autonomousapps/internal/asm.kt

```diff
@@ -430,18 +439,19 @@ private class AnnotationAnalyzer(
   private fun indent() = "  ".repeat(level)
 
   override fun visit(name: String?, value: Any?) {
-    fun getValue(value: Any?): String {
-      return if (value is String && value.contains("\n")) {
-        ""
-      } else {
-        value.toString()
+    val valueString = stringValueOfArrayElement(value)
+    log("${indent()}- AnnotationAnalyzer#visit: name=$name, value=(${value?.javaClass?.simpleName}, ${valueString})")
+
+    if (arrayName != null) {
+      arraySize++
+      if (valueString == "alias") {
+        isTypeAlias = true
       }
     }
 
-    log("${indent()}- AnnotationAnalyzer#visit: name=$name, value=(${value?.javaClass?.simpleName}, ${getValue(value)})")
     if (value is String) {
       METHOD_DESCRIPTOR_REGEX.findAll(value).forEach { result ->
-        addClass(result.value, ClassRef.Kind.NOT_ANNOTATION)
+        addClass(result.value, ClassRef.Kind.ANNOTATION)
```

Comment:
This was a bug. But fixing it broke TypeAliasSpec, because that was relying on analyzing the `@Metadata` annotation for detecting typealias usage.